data/sponsors.js

```diff
@@ -0,0 +1,20 @@
+// Multiplier is the value you need to multiply height by to get width
+export const GOLD_SPONSORS_ONE = [
+  { file: "Benevity.png", multiplier: 3.216, height: 90, website: "benevity.com" },
+  { file: "Hunter Hub.png", multiplier: 7.02, height: 70, website: "ucalgary.ca/hunter-hub" },
+];
+
+export const GOLD_SPONSORS_TWO = [
+  { file: "Arcurve.png", multiplier: 10, height: 50, website: "arcurve.com" },
+  { file: "Cisco Secure.jpg", multiplier: 3.41, height: 120, website: "cisco.com/site/us/en/products/security" },
```

Comment:
I tried going onto the cisco site but it doesn't work, maybe put in another link?

Reply:
Yeah not sure why it didn't work either - will just redirect to Cisco website